src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocContentLocationCheck.java

```diff
@@ -0,0 +1,243 @@
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
+
+/**
+ * <p>
+ * Checks that the Javadoc content begins from the same position
+ * for all Javadoc comments in the project. Any leading asterisks and spaces
+ * are not counted as the beginning of the content and are therefore ignored.
+ * </p>
+ * <p>
+ * It is possible to enforce two different styles:
+ * </p>
+ * <ul>
+ * <li>
+ * {@code First line} - Javadoc content starts from the first line:
+ * <pre>
+ * &#47;** Summary text.
+ *   * More details.
+ *   *&#47;
+ * public void method();
+ * </pre>
+ * </li>
+ * <li>
+ * {@code Second line} - Javadoc content starts from the second line:
+ * <pre>
+ * &#47;**
+ *   * Summary text.
+ *   * More details.
+ *   *&#47;
+ * public void method();
+ * </pre>
+ * </li>
+ * </ul>
+ * <p>
+ * This check does not validate the Javadoc summary itself nor its presence.
+ * The check will not report any violations for missing or malformed javadoc summary.
+ * To validate the Javadoc summary use
+ * <a href="https://checkstyle.org/config_javadoc.html#SummaryJavadoc">SummaryJavadoc</a> check.
+ * </p>
+ * <p>
+ * The <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html">
+ * Documentation Comment Specification</a> permits leading asterisks on the first line.
+ * For these Javadoc comments:
+ * </p>
+ * <pre>
+ * &#47;***
+ *   * Some text.
+ *   *&#47;
+ * &#47;************
+ *   * Some text.
+ *   *&#47;
+ * &#47;**           **
+ *   * Some text.
+ *   *&#47;
+ * </pre>
+ * <p>
+ * The documentation generated will be just "Some text." without any asterisks.
+ * Since these asterisks will not appear in the generated documentation,
+ * they should not be considered as the beginning of the Javadoc content.
+ * In such cases, the check assumes that the Javadoc content begins on the second line.
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code location} - Specify the policy on placement of the Javadoc content.
+ * Default value is {@code second_line}.
+ * </li>
+ * </ul>
+ * <p>
+ * By default Check validate that the Javadoc content starts from the second line:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocContentLocationCheck"/&gt;
+ * </pre>
+ * <p>
+ * This setting produces a violation for each multi-line comment starting
+ * on the same line as the initial asterisks:
+ * </p>
+ * <pre>
+ * &#47;** This comment causes a violation because it starts from the first line
+ *   * and spans several lines.
+ *   *&#47;
+ * &#47;**
+ *   * This comment is OK because it starts from the second line.
+ *   *&#47;
+ * &#47;** This comment is OK because it is on the single line. *&#47;
+ *</pre>
+ * <p>
+ * To ensure that Javadoc content starts from the first line:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocContentLocationCheck"&gt;
+ *   &lt;property name="location" value="FIRST_LINE"/&gt;
```

Comment:
I suggest to make property name consistent across javadoc, e.g. here it is `FIRST_LINE`, in line 91 it is `second_line`, in lines 41 and 50 it is `First line`

Reply:
done